pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd_migration_use-cases/guide.de-de.md

```diff
@@ -38,79 +43,41 @@ details[open]>summary::before {
 > This audit is mandatory before any possible migration to VCD on OVHcloud.
 >
 
-Welcome to this how-to guide to provide you with information on the process of migrating your vSphere/vCenter Hosted Private Cloud infrastructure to VMware Cloud Director in the VMware on OVHcloud ecosystem.
+This how-to guide will provide you with information on the process of migrating your Hosted Private Cloud maanaged vSphere/vCenter on OVHcloud services to a VMware Cloud Director on OVHcloud ecosystem.
```

Comment:
Typo: "maanaged" double "a".

"Hosted Private Cloud" is the group name of products that also include "Nutanix on OVHcloud". Use "VMware on OVHcloud" instead, which is the name of the VMware managed product we offer.

You don't need to specify "vSphere/vCenter".

Reply:
I was referencing "vCenter" for the purpose of make the user understand the cross vCenter migration.